packages/bitcore-wallet-client/lib/api.js

```diff
@@ -624,9 +624,14 @@ API.prototype.getBalanceFromPrivateKey = function(privateKey, coin, cb) {
   var B = Bitcore_[coin];
  
   var privateKey = new B.PrivateKey(privateKey);
-  var address = privateKey.publicKey.toAddress();
+
+  var address = privateKey.publicKey.toAddress().toString();
```

Comment:
you can use `toString(true)`  that will work for BTC and remove the prefix for BCH.